index.js

```diff
@@ -378,11 +378,27 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = null
+    var versionMsg = null
+
+    if (arguments.length === 1) {
+      version = opt
+      versionOpt = 'version'
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else if (arguments.length === 2) {
+      versionOpt = opt
+      version = msg
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else {
```

Comment:
I totally agree with the use-cases @bcoe listed above. And I would _love_ to be able to just use `.version()` with no args. :heart_eyes: